test/integration/expression-tests/image/coalesce/compare/test.json

```diff
@@ -1,12 +1,9 @@
 {
   "expression":
-  ["==", "missing",
```

Comment:
Do we still want this syntax to be covered by tests though? Otherwise we're reducing coverage for it, maybe both ways should be tested. Which would mean duplicating those tests instead of switching entirely.

Reply:
They are duplicated, that's what the new tests are for. (in this case "compare-verbose"). I'll make that more clear in the description!

Reply:
Ok all good then! I didn't realize that the new tests were for that.

Reply:
Currently this doubles the tests added in my previous PRs and probably adds more redundancy than is necessary. I haven't removed any yet, but perhaps we should consider removing some of these tests to lighten the CI burden.

Reply:
As long as these tests aren't flaky, I think it's ok.